AI/BattleAI/AttackPossibility.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * AttackPossibility.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+#include "common.h"
+#include "../../lib/BattleState.h"
+#include "../../lib/CCreatureHandler.h"
+#include "../../lib/spells/CSpellHandler.h"
+#include "../../lib/VCMI_Lib.h"
```

Comment:
I fix all your comments in last commit except this one, in this file: 
`#include "../../lib/BattleState.h"`
I hope that you will understand why ;).